CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [[Audit] Automatic integrity check with fingerprints back up](https://xen-orchestra.com/blog/xo-audit/) [#4844](https://github.com/vatesfr/xen-orchestra/issues/4844) (PR [#5077](https://github.com/vatesfr/xen-orchestra/pull/5077))
```

Comment:
Is there any doc or blog post about storing fingerprints on `www-xo`?

Reply:
`www-xo` is a public repository?

Reply:
Can we link to it from the changelog?

Reply:
It's already linked in the changelog entry.